.github/actions/deploy-lifecycle-manager-e2e/action.yml

```diff
@@ -109,7 +109,8 @@ runs:
         matrix.e2e-test == 'module-status-decoupling-with-deployment' ||
         matrix.e2e-test == 'purge-metrics' ||
         matrix.e2e-test == 'self-signed-certificate-rotation' ||
-        matrix.e2e-test == 'mandatory-module-metrics'}}  
+        matrix.e2e-test == 'mandatory-module-metrics' ||
+        matrix.e2e-test == 'mandatory-module-metrics-without-version-naming'}}
```

Comment:
I was a bit confused about the `without-version-naming` name and had to spend some time to figure out what this is about. Maybe we can rather call it `...with-old-naming-pattern`. Also not super happy with it because the name is actually not important in the processing, but it should be a bit clearer and it will get removed anyway.
Feel free to disagree, we can also keep it as you proposed.

Reply:
yes I was hesitanta lot about the name and even thought about old_naming_.. but then I thought I would just write the difference. but since you have the same thought I will rename it